Makefile

```diff
@@ -34,6 +34,9 @@ DOCKER_BUILDER_IMAGE_DIR ?= $(ROOT_DIR)/images/builder
 DOCKER_BUILDER_IMAGE_TAG ?= $(call files_checksum,$(DOCKER_BUILDER_IMAGE_DIR))
 DOCKER_BUILDER_IMAGE_FULL_NAME ?= $(DOCKER_REPO_NAME)/$(DOCKER_BUILDER_IMAGE_NAME):$(DOCKER_BUILDER_IMAGE_TAG)
 
+OPERATOR_IMAGE_RELEASE_NAME ?= kudo-spark-operator
+SPARK_IMAGE_RELEASE_NAME ?= spark
```

Comment:
given the nature of these variables, shall we default them to the full tags? e.g.
```
OPERATOR_IMAGE_RELEASE_TAG ?= $(DOCKER_REPO_NAME)/kudo-spark-operator:latest
SPARK_IMAGE_RELEASE_TAG ?= $(DOCKER_REPO_NAME)/spark:2.4.3-k8s-latest
```

And in the release job we will specify these tags fully. Another option would be to pass release tags only while keeping org/repo names static.

Reply:
I assume org and repo names will not change much, so I would rather go with the second option.